galvani/BioLogic.py

```diff
@@ -240,15 +253,27 @@ def MPTfileCSV(file_or_path):
     ]
 )
 
+VMPmodule_hdr_v2 = np.dtype(
+    [
+        ("shortname", "S10"),
+        ("longname", "S25"),
+        ("max length", "<u4"),
+        ("length", "<u4"),
+        ("version", "<u4"),
+        ("unknown2", "<u4"),  # 10 for set, log and loop, 11 for data
+        ("date", "S8"),
+    ]
+)
+
 # Maps from colID to a tuple defining a numpy dtype
 VMPdata_colID_dtype_map = {
     4: ("time/s", "<f8"),
     5: ("control/V/mA", "<f4"),
     6: ("Ewe/V", "<f4"),
-    7: ("dQ/mA.h", "<f8"),
+    7: ("dq/mA.h", "<f8"),
```

Comment:
I'm not sure I understand why this had to be renamed, but the knock-on effect is that old MPR files also get passed with this column name (presumably also the same with `<I>/mA` below, but I understand that one more). Is the point that ECLab itself uses the `dq` version of the name? Perhaps it would be good to add this in the release notes as the one "breaking change".

Reply:
Given that ID 23 is "dQ/mA.h" maybe this was always correct? I never understood what the difference is between `Q`, `q`, `<q>` etc.

Reply:
We should probably also clean up the comments on IDs 8 and 23 below since those lines are now unique.

Reply:
> Given that ID 23 is "dQ/mA.h" maybe this was always correct? I never understood what the difference is between `Q`, `q`, `<q>` etc.

Ah right, didn't notice that. I'm not sure of the difference beyond the fact that a file that contains dq used to parse it as dQ but now (correctly I guess) parses it as dq, so we should add the note.

Reply:
Just removed the comments; for the sake of everyone's time I think I'll merge this now without awaiting another round of reviews. @chatcannon of course you can have the final say before we release.

Reply:
Scratch that, apparently I don't have access to merging PRs either. @chatcannon could you do the honours (and maybe just remove the comments on top of this PR)?